config/config.go

```diff
@@ -35,7 +30,6 @@ type NodeConfig struct {
 	// parameters below are Rollkit specific and read from config
 	Aggregator         bool `mapstructure:"aggregator"`
 	BlockManagerConfig `mapstructure:",squash"`
-	DALayer            string `mapstructure:"da_layer"`
 	DAConfig           string `mapstructure:"da_config"`
```

Comment:
Nitpicking: `DAConfig` is a candidate for rename as it's just a server address right now.

Reply:
changed to `da_address` and using default (`:25560`)